src/common/utils/helpers.ts

```diff
@@ -68,7 +68,7 @@ export function mergeTransactions(
   payments: ConnectorTransaction[]
 ): ConnectorTransaction[] {
   const mergedTransactions = [...invoices, ...payments].sort((a, b) => {
-    return b.settleDate - a.settleDate;
+    return (b.settleDate ?? Date.now()) - (a.settleDate ?? Date.now());
```

Comment:
why using Date.now?

Reply:
i made settleDate optional as for failed and pending its null. so i have to add Date.now just to make ts happy for other connectors

Reply:
i was a bit wrong here. instead of making it optional it shall be number | null now. updated code accordingly